lib/spack/spack/package.py

```diff
@@ -1710,6 +1710,10 @@ def do_install(self, **kwargs):
             force (bool): Install again, even if already installed.
             install_deps (bool): Install dependencies before installing this
                 package
+            install_env_variables (bool): By default, the build environment
```

Comment:
I'm glad this will be the default - it's a bit unexpected for a tool to save your entire environment without you really explicitly asking for it.